core/types/confidential.go

```diff
@@ -89,6 +91,97 @@ func (tx *ConfidentialComputeRecord) setSignatureValues(chainID, v, r, s *big.In
 	tx.ChainID, tx.V, tx.R, tx.S = chainID, v, r, s
 }
 
+type ConfidentialComputeRequest2 struct {
+	EIP712    *eip712.EIP712TypedData `json:"eip712"`
+	Signature []byte                  `json:"signature"`
```

Comment:
Signature values in the record are deprecated because you are signing the EIP-712 message, not the record.